frontend/simple/js/state.js

```diff
@@ -75,6 +77,22 @@ const mutations = {
     if (!state.contracts[contractId] && !state.pending.includes(contractId)) {
       state.pending.push(contractId)
     }
+  },
+  addUser (state, {username, identity}) {
```

Comment:
Again, `identity` should be avoided. Either `identityContract` or `identityContractId`.